docs/classic/release-note-noble-thor-ea.rst

```diff
@@ -0,0 +1,93 @@
+
+Images
+------
+
+Ubuntu images can be downloaded from https://ubuntu.com/download/nvidia-jetson:
+
+
+* Ubuntu Server 24.04:
+
+  * https://people.canonical.com/~platform/images/nvidia-tegra/ubuntu-24.04-preinstalled-server-thor-arm64+jetson.img.xz
+  * Image SHA256SUM: ``6724ba86126228c989ae794bf2472a8109e3354fa7411becf089706a7e041a3e``
+
+* Boot firmware 38.2:
+
+  * https://developer.nvidia.com/downloads/embedded/L4T/r38_Release_v2.0/release/Jetson_Linux_R38.2.0_aarch64.tbz2
+  * Image SHA256SUM: ``bef352597d8ebcb1bedff1f2abef5957dbd34faa8c8cff7d9d1e576c18c7a46d``
+
+Hardware Platforms Tested
+-------------------------
+
+
+* Jetson AGX Thor Developer kit
+
+Release Highlights
+------------------
+
+
+* This is an early access release only targetting Jetson AGX Thor, it does not support the Jetson Orin development kits.
+* At this early stage, camera testing has been deferred to a later release.
+* The linux kernel used on the Jetson AGX Thor (`̀ linux-nvidia-tegra-ppadev-jetson``) under active development and has not yet been added to the official archive. Please note that this kernel is not stable, and future updates may introduce unexpected regressions.
+* Migrating from this Early Access image to a future release will require switching to a different kernel meta-package.
+
+
+Known issues
+------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Issue
+     - Description
+   * - `2122501 <https://bugs.launchpad.net/riverside/+bug/2122501>`_
+     - The Bluetooth controller firmware ``rtl8852cu_fw`` is currently missing during boot, which prevents the Bluetooth service from functioning correctly. This issue will be resolved in an upcoming update to the ``linux-firmware-nvidia-tegra`` package. As a workaround, :ref:`installing the nvidia-l4t-firmware package <classic/installation-noble:install nvidia proprietary software>` can resolve the issue immediately. Refer to the documentation here: .
+   * - `2121984 <https://bugs.launchpad.net/riverside/+bug/2121984>`_
+     - Connecting to a WPA3 802.11ax access point currently results in a kernel crash. This issue is still under investigation but can be avoided by `Updating the development kernel`_.
+   * - `2122572 <https://bugs.launchpad.net/riverside/+bug/2122572>`_
+     - The system currently exits suspend mode automatically after a few seconds, instead of waiting for a proper wake-up event (e.g., RTC alarm, keyboard input). This premature wake-up behaviour is a known issue and is expected to be resolved in an upcoming kernel update.
+   * - `2122629 <https://bugs.launchpad.net/riverside/+bug/2122629>`_
+     - Some of the stress tests performed are based on `stress-ng <https://github.com/ColinIanKing/stress-ng>`_: ``stress-ng --af-alg 0 --timeout 30 --oom-avoid-bytes 10% --skip-silent --verbose``. This issue is currently under investigation but can be avoided by `Updating the development kernel`_.
```

Comment:
Need to actually say what the issue is here, maybe something like

"some of the stress tests based on `stress-ng .....` fail with the message "FOO BAR". This issue is under ivnestigation"